### DISCUSSION TITLE
I'd like to make a warning go away

### DISCUSSION THREAD

**Discussion by GeorchW** [sqlalchemy/alembic#1389] [Usage Questions]
So, today I had a simple bug: When using ORM code that contains multi-line computed statements, they always generate warnings when being fed into Alembic. Like this:

```py
class MyModel(Base):
    some_column: Mapped[int] = mapped_column(Computed("""1
        + 2""")) # just a simple multi-line expression for demo purposes
    # makes more sense with longer expressions
```

Alembic will always complain with "UserWarning: Computed default on MyModel.start_holding cannot be modified".
I quickly looked into the code and found the culprit in `compare.py`:

```py
def _normalize_computed_default(sqltext: str) -> str:
    """we want to warn if a computed sql expression has changed.  however
    we don't want false positives and the warning is not that critical.
    so filter out most forms of variability from the SQL text.

    """

    return re.sub(r"[ \(\)'\"`\[\]]", "", sqltext).lower()
```

This code throws out whitespace characters for the comparison, but no newlines or tabs. Pretty easy fix!

```py
    return re.sub(r"[ \(\)'\"`\[\]\t\r\n]", "", sqltext).lower()
```

This seems to work, and I'd like to provide this back to the upstream, but tbh, I have no idea how the testing works here, so I can't write a PR. I couldn't find any existing tests for this code, but maybe I didn't look hard enough. 

Anyway, investing more time into this is not worth the effort to get rid of a couple warnings, so I'm going to stop here.  The fix is already here, feel free to commit it.

**Comment by CaselIT** [comment]
Hi,

I'm not sure what is different from other projects. The workflow the classic open an issue, and if the fix is trivial provide a PR (maybe in draft) and ask for questions if something is unclear like tests.

Searching for computed in the code base shows this test module that's dedicated to computed https://github.com/sqlalchemy/alembic/blob/main/alembic/testing/suite/test_autogen_computed.py

**Reply by GeorchW** [reply]
Thank you for the hint. Sorry for being a bit harsh on the words earlier. The process seems to be quite strict to me to apply to even very trivial changes. 

Anyway, the issue is here: https://github.com/sqlalchemy/alembic/issues/1391
I'll add the MR shortly.
